src/resources/Client.ts

```diff
@@ -1,26 +1,46 @@
 import camelcaseKeys from 'camelcase-keys';
 import filterKeys from '../utils/Filter';
+import associationDefaults from "../utils/Associations";
 
+import { Association } from "./Enums";
 import type { ClientJSON } from './JSON';
 import type { ClientProps } from './Props';
 
-interface ClientPayload extends ClientProps {};
+import { Session } from "./Session";
+
+interface ClientAssociations {
+  sessions: Session[];
+}
+
+interface ClientPayload extends ClientProps, ClientAssociations {};
 
 export interface Client extends ClientPayload {};
 
 export class Client {
-  static attributes = ['id', 'sessionIds', 'signUpAttemptId', 'signInAttemptId',
-    'lastActiveSessionId', 'lastActiveSessionId', 'createdAt', 'updatedAt'];
+  static attributes = ['id', 'sessionIds', 'sessions', 'signUpAttemptId', 'signInAttemptId',
```

Comment:
🔨 The attribute list contains only native fields, not associations, thus `sessions` shouldn't be in there.

Reply:
Ah, I see! Cool, thanks for letting me know. I've removed the `sessions` attribute from this list.